File: apicsharpfacturas/Models/BillDetailEntity.cs


using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace apicsharpfacturas.Models
{
    public class BillDetailEntity {

        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int id { get; set; }
        [JsonIgnore]
        public BillEntity? billEntity { get; set; }
        public ProductEntity product { get; set; }
        public int quantity { get; set; }
        public double unitValue { get; set; }
        public double totalValue { get; set; }
    } 
}

File: apicsharpfacturas/ClientApp/src/app/models/BillDetailEntity.ts
import { BillEntity } from "./BillEntity";
import { ProductEntity } from "./ProductEntity";



export class BillDetailEntity{

    public id?: number;
    public BillEntity?: BillEntity;
    public product?: ProductEntity;
    public quantity?: number;
    public unitValue?: number;
    public totalValue?: number;

}